crates/turborepo-lib/src/shim.rs

```diff
@@ -338,12 +359,79 @@ fn is_turbo_binary_path_set() -> bool {
     env::var("TURBO_BINARY_PATH").is_ok()
 }
 
+fn init_env_logger(verbosity: usize) {
+    // configure logger
+    let level = match verbosity {
+        1 => LevelFilter::Info,
+        2 => LevelFilter::Debug,
+        3 => LevelFilter::Trace,
+        _ => LevelFilter::Warn,
```

Comment:
These are set to match https://turbo.build/repo/docs/reference/command-line-reference#-v--vv--vvv